core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -166,6 +175,7 @@ function (string $value): string {
                             ->defaultValue(false)
                         ->end()
                         ->arrayNode('sizes')
+                            ->info('This allows to define image sizes directly in the configuration, rather than just via the tl_image_size table.')
```

Comment:
Suggestion: *This allows to define image sizes directly in the configuration in addition to the Contao back end (tl_image_size table).*